lib/unit-ruby/util/connection.rb

```diff
@@ -24,7 +24,11 @@ def initialize
     #
     # @return the resource (or array of resources) returned from the API
     def get(path, params = nil)
-      response = connection.get(path, params)
+      response = connection.get do |req|
+        req.url path
+        req.headers['Content-Type'] = 'application/vnd.api+json'
+        req.params = params if params
```

Comment:
Is there something about these newly added endpoints where the method signature of params=nil no longer works?

Reply:
Nope, I think I was just replicating the other methods.